### PR TITLE
 websocket: don't clone buffer

### PR DESCRIPTION
**Pull request by tsctx**
No clones are needed here...

### CODE REVIEW

File: lib/web/websocket/websocket.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ class WebSocket extends EventTarget {
       // not throw an exception must increase the bufferedAmount attribute
       // by the length of data’s buffer in bytes.
 
-      const ab = new FastBuffer(data, data.byteOffset, data.byteLength)
+      const ab = new FastBuffer(data.buffer, data.byteOffset, data.byteLength)
```
Comment (tsctx) [inline lib/web/websocket/websocket.js:290]:
```suggestion
      const ab = new FastBuffer(data.buffer.slice(data.byteOffset, data.byteOffset + data.byteLength), 0, data.byteLength)
```

File: lib/web/websocket/websocket.js [hunk 2]
```diff
@@ -287,7 +287,7 @@ class WebSocket extends EventTarget {
       // not throw an exception must increase the bufferedAmount attribute
       // by the length of data’s buffer in bytes.
 
-      const ab = new FastBuffer(data, data.byteOffset, data.byteLength)
+      const ab = new FastBuffer(data.buffer.slice(data.byteOffset, data.byteOffset + data.byteLength), 0, data.byteLength)
```
Comment (tsctx) [inline lib/web/websocket/websocket.js:290]:
```suggestion
      const ab = new FastBuffer(data.buffer, data.byteOffset, data.byteLength)
```

**Comment by tsctx** [comment]
We have improved performance, but it is really a bug fix.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**